js/packages/cli/src/helpers/schema.ts

```diff
@@ -0,0 +1,378 @@
+import { BinaryReader, BinaryWriter } from 'borsh';
+import base58 from 'bs58';
+import { PublicKey } from '@solana/web3.js';
+type StringPublicKey = string;
+
+import BN from 'bn.js';
+const EDITION_MARKER_BIT_SIZE = 248;
+
+enum MetadataKey {
+  Uninitialized = 0,
+  MetadataV1 = 4,
+  EditionV1 = 1,
+  MasterEditionV1 = 2,
+  MasterEditionV2 = 6,
+  EditionMarker = 7,
+}
+
+class MasterEditionV1 {
```

Comment:
This should be MasterEditionV2, not V1

Reply:
You're definitely using the wrong struct (the older version) given the master edition creation command being used above. Wont match.

Reply:
Ok, those weren't used anyway, I was maintaining the SHEMA structure used by the front-end
I removed them for now!